bigbone/src/main/kotlin/social/bigbone/api/method/StatusMethods.kt

```diff
@@ -118,7 +121,8 @@ class StatusMethods(private val client: MastodonClient) {
         mediaIds: List<String>? = null,
         sensitive: Boolean = false,
         spoilerText: String? = null,
-        language: String? = null
+        language: String? = null,
+        addIdempotencyKey: Boolean = true
```

Comment:
This should be of type `Boolean?` to ensure Java method overload generation works as expected.

Reply:
Isn't Jvm overloading based on a parameter's default value, not its nullability, though?

For example, `client.statuses().postStatus("content text")` is already a valid way to post a status from Java, although `sensitive` is not nullable.

Reply:
Ah, correct. I was wrong. Thx!